src/frontend/css/boards.css

```diff
@@ -0,0 +1,46 @@
+/* svg */
+.blue cg-board {
+  background-image:url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4PSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIgogICAgIHZpZXdCb3g9IjAgMCA4IDgiIHNoYXBlLXJlbmRlcmluZz0iY3Jpc3BFZGdlcyI+CjxnIGlkPSJhIj4KICA8ZyBpZD0iYiI+CiAgICA8ZyBpZD0iYyI+CiAgICAgIDxnIGlkPSJkIj4KICAgICAgICA8cmVjdCB3aWR0aD0iMSIgaGVpZ2h0PSIxIiBmaWxsPSIjZGVlM2U2IiBpZD0iZSIvPgogICAgICAgIDx1c2UgeD0iMSIgeT0iMSIgaHJlZj0iI2UiIHg6aHJlZj0iI2UiLz4KICAgICAgICA8cmVjdCB5PSIxIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIiBmaWxsPSIjOGNhMmFkIiBpZD0iZiIvPgogICAgICAgIDx1c2UgeD0iMSIgeT0iLTEiIGhyZWY9IiNmIiB4OmhyZWY9IiNmIi8+CiAgICAgIDwvZz4KICAgICAgPHVzZSB4PSIyIiBocmVmPSIjZCIgeDpocmVmPSIjZCIvPgogICAgPC9nPgogICAgPHVzZSB4PSI0IiBocmVmPSIjYyIgeDpocmVmPSIjYyIvPgogIDwvZz4KICA8dXNlIHk9IjIiIGhyZWY9IiNiIiB4OmhyZWY9IiNiIi8+CjwvZz4KPHVzZSB5PSI0IiBocmVmPSIjYSIgeDpocmVmPSIjYSIvPgo8L3N2Zz4K')
+}
```

Comment:
Why do some of these use image files and others some weird SVG hash? Why can't they all just image files?

Reply:
[lichess pieces-css](https://github.com/lichess-org/lila/tree/master/public/piece-css) as well as chessground use this SVG as Base64 code. I assumed it was the slicker and potentially lighter-weight way of doing SVG. So I ran with it.

It all could be simple image files. When it comes to the boards with textures they have to be either PNG or JPG, so if we wanted to be totally consistent that would make sense.

I used this [SVG to Base64 converter](https://base64.guru/converter/encode/image/svg) to convert things back and forth.

Reply:
Apparently, it's called [inlining images](https://www.argentinaindependent.com/2018/08/12/inline-images.html), and it's mostly a relic of the past.

TL;DR - It makes the images load with the page, all at once. Without doing this, it's possible for the background to load first, then the pieces. This makes the image a component of the webpage, so it only loads when everything is ready together.

Reply:
I can see that being useful for a chessboard - only displaying the board when everything is ready.